README.md

```diff
@@ -24,3 +24,25 @@ The SDK is tested against these versions of `esp-idf`:
 Previous versions of `esp-idf` are not guaranteed to work. If you find a problem using a later
 version of `esp-idf`, please [open an
 issue](https://github.com/astarte-platform/astarte-device-sdk-esp32/issues).
+
+## Notes on non volatile memory (NVM)
+
+The device's Astarte credentials are always stored in the NVM. This means that credentials will
+be preserved in between reboots.
+Two storage methods can be used for the credentials. A FAT32 file system or the standard non
+volatile storage (NVS) library provided by the ESP32. The FAT32 is the default storage choice as it
+provides better wear levelling functionality.
+As a side effect of NVM usage, credentials will be preserved also between device flashes using
```

Comment:
Add something like :

If you want to use NVS storage, add the  `astarte_credentials_use_nvs_storage` function , before initializing astarte_credentials.
``` c
astarte_credentials_use_nvs_storage(NVS_PARTITION);
astarte_credentials_init();
 ```